src/lazy/expanded/macro_evaluator.rs

```diff
@@ -1228,6 +1272,62 @@ mod tests {
         )
     }
 
+    #[test]
+    fn make_sexp() -> IonResult<()> {
+        eval_template_invocation(
+            r#"(macro foo ($x) (make_sexp [1, $x, 3] [(literal a), $x, (literal c)]))"#,
+            r#"
+                (:foo 5)
+                (:foo quuz)
+                (:foo {a: 1, b: 2})
+                (:foo [])
+            "#,
+            r#"
+                (1 5 3 a 5 c)
```

Comment:
I expected `(:foo 5)` to produce `([1, 5, 3] [a, 5, c])`. Is `make_sexp` supposed to flatten its arguments?

Reply:
Offline discussion—I was reminded that `make_list`, etc. are to act like `make_string` in that it concatenates the _content_ of the arguments.